market/src/contract.rs

```diff
@@ -29,19 +29,28 @@ impl Market {
         }
     }
 
-    pub fn get_data_data(&self) -> MarketData{
+    pub fn get_data_data(&self) -> MarketData {
         self.market.clone()
     }
 
-    pub fn is_published(&self) -> bool{
+    pub fn get_proposals(&self) -> Vec<u64> {
+        self.proposals.clone()
+    }
+
+    pub fn is_published(&self) -> bool {
         self.published
     }
 
-    pub fn is_resolved(&self) -> bool{
+    pub fn is_resolved(&self) -> bool {
         self.resolved
     }
```

Comment:
This getters should go in `views.rs`

cc @susguzman

Reply:
Um, it could be. I will move them!